windows/ReactTestApp/Manifest.cpp

```diff
@@ -19,6 +21,65 @@ namespace
             return std::nullopt;
         }
     }
+
+    std::any getAny(const nlohmann::json &j)
+    {
+        switch (j.type()) {
+            case value_t::null:
+                return std::nullopt;
+                break;
+            case value_t::boolean:
+                return j.get<boolean>();
+                break;
+            case value_t::number_integer:
+                return j.get<std::int64_t>();
+                break;
+            case value_t::number_unsigned:
+                return j.get<std::uint64_t>();
+                break;
+            case value_t::number_float:
+                return j.get<double>();
+                break;
+            case value_t::string:
+                return j.get<std::string>();
+                break;
+            case value_t::object: {
+                std::map<std::string, std::any> map;
+                for (auto &&e : j.items()) {
+                    map.insert(std::pair(e.key(), getAny(e.value())));
+                }
+                return map;
+                break;
+            }
+            case value_t::array: {
+                std::vector<std::any> array;
+                for (auto &&e : j.items()) {
+                    array.push_back(getAny(e.value()));
+                }
+                return array;
+                break;
+            }
+            // value_t::discarded. This case should never be hit since the check for malformed json
+            // is done previously in GetManifest()
+            default:
+                return nullptr;
```

Comment:
If this case is not reachable, we should add an assertion (and return a proper value):
```suggestion
            default:
                assert(false);
                return std::nullopt;
```